sdk/tests/deqp/framework/opengl/gluDrawUtil.js

```diff
@@ -40,7 +40,7 @@ var gluShaderProgram = framework.opengl.gluShaderProgram;
  */
 gluDrawUtil.VertexArrayBinding = function(type, location, components, elements, data, stride, offset) {
     this.type = type;
-    this.location = location === undefined ? -1 : location;
+    this.location = location === undefined ? 0 : location;
```

Comment:
That's what I would expect. Please undo this change and the change to bindingPointFromName. Thanks.